decidim-debates/spec/system/debates_dates.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Date comprober", versioning: true, type: :system do
```

Comment:
"Date comprober" 🥇 👍🏽 Well, actually, it is a good practice for the name of the spec to match the file name!

```suggestion
describe "Debate dates", versioning: true, type: :system do
```